src/go/rpk/pkg/tuners/disk_checkers.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 func CreateDirectoryCheckers(
-	fs afero.Fs,
+	_ afero.Fs,
```

Comment:
drop arg here, remove from callsite (here and the rest of this file) ?

or is this for an interface implementation?

Reply:
It wasn't an interface implementation, removed 👍